happypose/pose_estimators/cosypose/cosypose/datasets/bop.py

```diff
@@ -6,15 +6,13 @@
 import numpy as np
 import pandas as pd
 import torch
+from bop_toolkit_lib import inout
 from PIL import Image
 
-from happypose.pose_estimators.cosypose.cosypose.config import BOP_TOOLKIT_DIR, MEMORY
+from happypose.pose_estimators.cosypose.cosypose.config import MEMORY
 from happypose.pose_estimators.cosypose.cosypose.utils.logging import get_logger
 from happypose.toolbox.lib3d.transform import Transform
 
-sys.path.append(str(BOP_TOOLKIT_DIR))
-from bop_toolkit_lib import inout
-
 sys.path = sys.path[:-1]
```

Comment:
while here, do you know what is this about ? I'd be glad to remove this monstruosity :)

Reply:
I'm talking about `sys.path = sys.path[:-1]` : we should avoid all `sys.path` use and manipulation inside the software.

Reply:
I agree! Just pushed a commit

Reply:
thanks ❤️